tests/behat/features/ra_bug-pieter.feature

```diff
@@ -0,0 +1,33 @@
+Feature: A RA manages tokens tokens registered in the selfservice portal
+  In order to manage tokens
+  As a RA
+  I must be able to manage second factor tokens in RA
+
+  Scenario: Provision an institution and a user to promote later on by an authorized institution
+    Given institution "institution-a.example.com" can "select_raa" from institution "institution-a.example.com"
+      And institution "institution-a.example.com" can "use_ra" from institution "institution-a.example.com"
+      And institution "institution-d.example.com" can "use_raa" from institution "institution-a.example.com"
+
+      And institution "institution-f.example.com" can "select_raa" from institution "institution-f.example.com"
+      And institution "institution-f.example.com" can "use_ra" from institution "institution-a.example.com"
+      And institution "institution-f.example.com" can "use_ra" from institution "institution-b.example.com"
```

Comment:
Why mention institution-b if it is not used in the scenario? Idem for line 19

Reply:
This was in order to initially reproduce the bug report as close as possible. Removed it for now.